nengo_gui/page.py

```diff
@@ -238,6 +239,10 @@ def execute(self, code):
                 self.error = dict(trace='must declare a nengo.Network '
                                   'called "model"', line=line)
             model = None
+        else:
+            # set all the seeds so that creating components doesn't affect
+            #  the neural model itself
+            nengo_gui.seed_generation.define_all_seeds(model)
```

Comment:
Ah, very good points.  Yes, it should get moved back to the build, which is where I had it first...  That'll just be removing commit a452c32